src/common/options/global.yaml.in

```diff
@@ -2920,8 +2920,8 @@ options:
   level: advanced
   desc: The maximum interval seconds for update pg's reported_epoch,
         benefit for osdmap trim when osdmap not change frequently.
-  default: 500
   with_legacy: true
+  default: 5
```

Comment:
How did we get away with 500 before https://github.com/ceph/ceph/pull/54491? Was it the removed usage of `need_publish`?

---

> default configuration option for the manager collection of the OSD data.

Can you please share the option name?

Reply:
> How did we get away with 500 before #54491? Was it the removed usage of `need_publish`?

So it seems. I am still comparing logs for both versions.

Reply:
> Can you please share the option name?

mgr_stats_period